src/Sulu/Bundle/AdminBundle/Resources/js/containers/List/fieldFilterTypes/TextFieldFilterType.js

```diff
@@ -21,10 +21,6 @@ class TextFieldFilterType extends AbstractFieldFilterType<?{eq: string}> {
     }
 
     getValueNode(value: ?{eq: string}) {
-        if (!value) {
```

Comment:
Do you remember why you implemented this previously? I suspect there is a similar problem in the `DropdownFieldFilterType` and maybe in the `DateTimeFieldFilterType`

Reply:
I think that is just a leftover from before I was using a Promise here... I'll check the other types as well.